### PR TITLE
:tada: Source Looker: Migrate to native CDK

### LINKED ISSUE #4353: Add unit test for "Look" in Looker source
## Tell us about the problem you're trying to solve
- Add follow up tests for #3911.

## Describe the solution you’d like
- We need to create a "Look" in our looker sandbox and reference its ID in our tests for this feature to be tested consistently.

## Describe the alternative you’ve considered or used
N/A

## Additional context
N/A

## Are you willing to submit a PR?
N/A

### PR DESCRIPTION
**Pull request by antixar**
## What
Refactoring and migration to the latest CDK version.

## How
New features:
- All stream have been migrated to the latest relevant CDK
- All static schemas are generated dynamically with the Looker Swagger file
- All stream should be tested by integration tests.
- the query_history stream supports incremental mode.

## Recommended reading order
1. `source.py`
2. `streams.py`

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

   
#### Community member or Airbyter
   
- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated 
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [x] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [x] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [x] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [x] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [x] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

### CODE REVIEW

File: airbyte-integrations/connectors/source-looker/integration_tests/configured_catalog.json [hunk 1]
```diff
@@ -3,372 +3,11882 @@
     {
       "stream": {
         "name": "color_collections",
-        "json_schema": {},
-        "supported_sync_modes": ["full_refresh"],
-        "source_defined_cursor": false
+        "json_schema": {
```
Comment (tuliren) [inline airbyte-integrations/connectors/source-looker/integration_tests/configured_catalog.json]: I think we no longer need to write out the `json_schema` in the `configured_catalog.json` file.

Removing these schemas could make the PR much cleaner. Also we don't need to update the code in two places if there is any change in the schema.

File: airbyte-integrations/connectors/source-looker/integration_tests/configured_run_looks_catalog.json [hunk 2]
```diff
@@ -0,0 +1,233 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "run_looks",
+        "json_schema": {
```
Comment (tuliren) [inline airbyte-integrations/connectors/source-looker/integration_tests/configured_run_looks_catalog.json:6]: Same here. The `json_schema` property can be empty `{}`.

File: airbyte-integrations/connectors/source-looker/source_looker/streams.py [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+        record_query_id = latest_record["query_id"]
+        if not self._is_finished and self._last_query_id == record_query_id:
+            if not self._last_query_id:
+                self._last_query_id = record_query_id
+            return current_stream_state
+        return {self.cursor_field: max((current_stream_state or {}).get(self.cursor_field) or "", latest_record[self.cursor_field])}
+
+
+class RunLooks(LookerStream):
+    """ "
```
Comment (tuliren) [inline airbyte-integrations/connectors/source-looker/source_looker/streams.py:422]: Nitpick. Extra quotation mark here.

```suggestion
    """
```

**Review by grubberr — APPROVED** [review]

File: airbyte-integrations/connectors/source-looker/integration_tests/configured_catalog.json (hunk 1 above)
Comment (antixar) [reply airbyte-integrations/connectors/source-looker/integration_tests/configured_catalog.json:6]: these files were generated by my separate script. I've fixed. Now they should include empty json_schemas only.

**Review by tuliren — COMMENTED** [review]
> these files were generated by my separate script. I've fixed. Now they should include empty json_schemas only.

@antixar, have you pushed the updated `configured_catalog.json` and `configured_run_looks_catalog.json`? The `json_schema` is still fully populated.

**Comment by antixar** [comment]
@tuliren , you are right. It was my mistake. Please check again

**Review by tuliren — APPROVED** [review]
Cool. Thanks for the update.

**Pull request merged**